pkg/tempopb/tempo.proto

```diff
@@ -46,6 +49,71 @@ service Metrics {
   rpc QueryRange(QueryRangeRequest) returns (QueryRangeResponse) {}
 }
 
+service BackendScheduler {
```

Comment:
maybe add to a new proto file in the same folder? scheduler.proto or something? wdyt?

Reply:
Simple enough, I've updated.